go.mod

```diff
@@ -5,6 +5,7 @@ go 1.22.0
 toolchain go1.22.5
 
 replace (
+	github.com/rancher/rke => github.com/rancher/rke v1.6.0-rc10
```

Comment:
@kinarashah can you elaborate on why you've added a replace directive for this import? I'd like to avoid adding new replacement directives wherever possible.

Reply:
@MbolotSuse I understand the need to avoid replacement directives, note that this is needed only for RCs and will be removed in the final version after RKE is released. 

The reason behind this is that we don't tag RKE RCs with strict semver versioning, we need them to be dot separated (`rc.10` in this case and we'd be changing our RC tags in future). Without dots, go mod considers `rc10` < `rc9` so it doesn't pick the latest RC and stays at `rc9`. Once release is tagged, we can remove this line when we bump it to `v1.6.0`.

Reply:
Alright, please make sure to file the follow-up PR before the release goes out.